File: docs/sf_objects_collection/fix.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'sf_objects.html');
let content = fs.readFileSync(filePath, 'utf8');

content = content.replace(/\\`/g, '`').replace(/\\\$/g, '$');

fs.writeFileSync(filePath, content, 'utf8');

File: docs/sf_objects_collection/fix.py
import os

path = r'c:\Users\121pi\Documents\myHP\MyCyberHP\docs\sf_objects_collection\sf_objects.html'
with open(path, 'r', encoding='utf-8') as f:
    c = f.read()

c = c.replace('\\`', '`').replace('\\$', '$')

with open(path, 'w', encoding='utf-8') as f:
    f.write(c)
